lib/DataView.js

```diff
@@ -143,15 +143,15 @@ DataView.prototype.refresh = function () {
  *
  * Where:
  *
- * {Number | String} id         The id of an item
- * {Number[] | String{}} ids    An array with ids of items
+ * {number | string} id         The id of an item
+ * {number[] | string{}} ids    An array with ids of items
  * {Object} options             An Object with options. Available options:
- *                              {String} [type] Type of data to be returned. Can
+ *                              {string} [type] Type of data to be returned. Can
  *                                              be 'DataTable' or 'Array' (default)
- *                              {Object.<String, String>} [convert]
- *                              {String[]} [fields] field names to be returned
+ *                              {Object.<string, string>} [convert]
+ *                              {string[]} [fields] field names to be returned
  *                              {function} [filter] filter items
- *                              {String | function} [order] Order the items by
+ *                              {string | function} [order] Order the items by
```

Comment:
I'm not sure if this will work. Look up 'Documenting a parameter's properties' in the [reference](http://usejsdoc.org/tags-param.html) for a valid way to do this. But again, my jsdoc-savvy is limited. Feel free to convince me this is OK.

Reply:
Ah, I hadn't seen that! Thank you - will revise.

Reply:
I'm assuming here you'll do it in a later iteration, which is perfectly fine.

Reply:
This one's particularly nasty because the function takes `args`

Reply:
OK, this sounds like a job for `@callback`.